packages/ui/src/molecules/QuantitySelector/QuantitySelector.tsx

```diff
@@ -0,0 +1,77 @@
+import type { InputHTMLAttributes } from 'react'
+import React from 'react'
+
+import type { InputProps } from '../../atoms/Input'
+import Input from '../../atoms/Input'
+import type { IconButtonProps } from '../IconButton'
+import IconButton from '../IconButton'
+
+export interface QuantitySelectorProps
+  extends InputHTMLAttributes<HTMLDivElement> {
+  /**
+   * Sets the current value that should be displayed on the input at the center of the quantity selector.
+   */
+  quantity: number | string
+
+  /**
+   * Object with properties that will be passed forward the inner IconButton atom at the left of the input.
+   */
+  leftButtonProps: IconButtonProps
+
+  /**
+   * Object with properties that will be passed forward the inner Input atom between the two buttons.
+   */
+  inputProps: InputProps
+
+  /**
+   * Object with properties that will be passed forward the inner IconButton atom at the right of the input.
+   */
+  rightButtonProps: IconButtonProps
+
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   *
+   * @default 'store-quantity-selector'
+   */
+  testId?: string
+}
+
+const QuantitySelector = ({
+  quantity,
+  testId = 'store-quantity-selector',
+  leftButtonProps,
+  inputProps,
+  rightButtonProps,
+  ...otherDivProps
```

Comment:
There has been [some past discussion on Slack](https://vtex.slack.com/archives/C01SMBU12J3/p1634561613180400) to name the rest params as `...otherProps`. I understand you were probably not aware of it but thought I should mention!

Reply:
At first I thought that it would be better to make it explicit to which element the props were going to, but it's better to keep the pattern, for the sake of consistency. Fixed at https://github.com/vtex/faststore/pull/1083/commits/8e2eafdd31bc25d61f95f3a0dc5a5b854a95b768